golang/cosmos/package.json

```diff
@@ -39,6 +38,18 @@
       "@agoric"
     ]
   },
+  "files": [
+    "Makefile*",
+    "app",
+    "binding.gyp.in",
```

Comment:
Is this platform-dependent generated code?

Reply:
> Is this platform-dependent generated code?

Nope, it's a template file used by the Makefile.